src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -39,31 +40,66 @@
 			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
 
-		<!--  Titlebar  -->
-		<Border x:Name="TitlebarArea" Height="40">
+		<!--  Titlebar Area  -->
+		<Grid x:Name="TitlebarAreaGrid">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="Auto" />
+				<ColumnDefinition Width="*" />
+			</Grid.ColumnDefinitions>
+
+			<Button
+				x:Name="TitlebarAreaBackButton"
+				Width="40"
+				Height="32"
+				Margin="4,4,0,0"
+				Background="Transparent"
+				BorderThickness="0"
+				Click="TitlebarAreaBackButton_Click"
+				IsEnabled="{x:Bind MainContentFrame.CanGoBack, Mode=OneWay}">
+				<Button.Resources>
+					<ResourceDictionary>
+						<StaticResource x:Key="ButtonBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
+					</ResourceDictionary>
+				</Button.Resources>
+				<AnimatedIcon x:Name="BackAnimatedIcon">
+					<AnimatedIcon.Source>
+						<animatedvisuals:AnimatedBackVisualSource />
+					</AnimatedIcon.Source>
+					<AnimatedIcon.FallbackIconSource>
+						<SymbolIconSource Symbol="Back" />
+					</AnimatedIcon.FallbackIconSource>
+				</AnimatedIcon>
+			</Button>
+
 			<TextBlock
-				Padding="16,0"
+				x:Name="TitlebarAreaTitle"
+				Grid.Column="1"
+				Margin="4,0,4,-2"
 				VerticalAlignment="Center"
 				Style="{StaticResource BodyStrongTextBlockStyle}"
 				Text="{helpers:ResourceString Name=Properties}" />
-		</Border>
+
+		</Grid>
 
 		<!--  Main NavigationView  -->
 		<NavigationView
 			x:Name="MainPropertiesWindowNavigationView"
 			Grid.Row="1"
 			IsBackButtonVisible="Collapsed"
-			IsPaneToggleButtonVisible="False"
+			IsPaneToggleButtonVisible="True"
```

Comment:
Toggle button shouldn't be visible

Reply:
We don't need toggle button? I thought it was needed to let user decide if remain navview pane opened or closed

Reply:
Depends on the placement of the toggle button, it's alright if it's in the title bar but otherwise it just wastes space.